bin/rucio

```diff
@@ -1403,9 +1403,9 @@ def update_rule(args):
     if args.lifetime:
         options['lifetime'] = None if args.lifetime.lower() == "none" else int(args.lifetime)
     if args.locked:
-        if args.locked == "True":
+        if args.locked.title() == "True":
             options['locked'] = True
-        elif args.locked == "False":
+        elif args.locked.title() == "False":
```

Comment:
I also noticed a possible improvement in the code, we should abort on an invalid input instead of silently continuing:
```python
if args.locked:
  if args.locked.lower() not in ("true", "false"):
    logger.error('--locked should be either "true" or "false"')
    return FAILURE
  options['locked'] = args.locked.lower() == "true"
```